From: Maximiliano Curia <maxy@gnuservers.com.ar>
Date: Thu, 6 Sep 2018 13:03:07 +0200
Subject: Drop the patch version for breeze

Not every plasma update includes changes in breeze, and it's expected to be "stable" during the
plasma minor lifecycle.
---
 CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0a769e0..bb50c4d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -122,7 +122,9 @@ add_feature_info("XorgLibinput" XORGLIBINPUT_FOUND "Libinput driver headers need
 
 include(ConfigureChecks.cmake)
 
-find_package(Breeze ${PROJECT_VERSION} CONFIG)
+string(REGEX MATCHALL "[0-9]+" PROJECT_VERSION_COMPONENTS ${PROJECT_VERSION})
+list(GET PROJECT_VERSION_COMPONENTS 1 PROJECT_VERSION_MINOR)
+find_package(Breeze ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR} CONFIG)
 set_package_properties(Breeze PROPERTIES
                        TYPE OPTIONAL
                        PURPOSE "For setting the default window decoration plugin")
